Formula/forego.rb

```diff
@@ -1,8 +1,8 @@
 class Forego < Formula
   desc "Foreman in Go"
   homepage "https://github.com/ddollar/forego"
-  url "https://github.com/ddollar/forego/archive/v0.16.1.tar.gz"
-  sha256 "d4c8305262ac18c7e51d9d8028827f83b37fb3f9373d304686d084d68033ac6d"
+  url "https://github.com/ddollar/forego/archive/20170327195458.tar.gz"
```

Comment:
Is upstream not using semver? We generally want to avoid date based versioning in new formulae. From a semver perspective this basically implies `0.0.20170327195458`.